roles/container_runtime/tasks/package_docker.yml

```diff
@@ -137,6 +137,11 @@
   changed_when: False
   register: r_docker_already_running_result
 
+- name: Fixup SELinux permissions for docker
+  shell: |
+           semanage fcontext -a -e /var/lib/docker/overlay2 "{{ docker_storage_path }}/overlay2"
+           restorecon -R -v "{{ docker_storage_path }}/overlay2"
+
```

Comment:
Lets still check for the presence of {{ docker_storage_path }}/overlay2 before triyng to muck with it.
```
- name: Check for docker_storage_path/overlay2
  stat:
    path: "{{ docker_storage_path }}/overlay2"
  register: docker_storage_path

- name: Fixup SELinux permissions for docker
  shell: |
           semanage fcontext -a -e /var/lib/docker/overlay2 "{{ docker_storage_path }}/overlay2"
           restorecon -R -v "{{ docker_storage_path }}/overlay2"
  when: docker_storage_path.stat.exists
```

Reply:
Didn't we remove something like this recently?

Reply:
Yes, because we thought it no longer necessary, but it seems like it may be under certain circumstances now.